stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -726,6 +725,18 @@ public protocol _TensorArrayProtocolEnhanced : TensorArrayProtocol {
     where C.Element == CTensorHandle
 }
 
+
+extension _TensorArrayProtocolEnhanced {
+  func _dtypes: [TF_DataType] = {
```

Comment:
`var`, not `func`. No `=` necessary. Also, this is not used outside of this file, so make the extension be a `private extension`.

Reply:
> `var`, not `func`. No `=` necessary. Also, this is not used outside of this file, so make the extension be a `private extension`.

Ah, it was a typo. Thanks for catching it.